### DISCUSSION TITLE
Building a dashboard style UI with Wave

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#1773] [Q&A] [answered]
Hi!I am building a dashboard style web app using Wave and need help with the following:
1. How do I add multiple stat cards in a single page to mimic a dashboard style grid layout?
2. If stat cards are not clickable (since they don't have a `name` attribute), is there any way to link it to another tab in order to redirect the user?

**Comment by vopani** [comment, marked as answer]
1. https://wave.h2o.ai/docs/examples/dashboard and https://github.com/h2oai/wave/tree/master/py/demo
2. https://github.com/h2oai/wave/issues/945#issuecomment-895065239

**Reply by n-srinidhi** [reply]
@vopani Thank you for your response! This is super helpful. Regarding the form card, Using a simple Form card would mean not getting a fancy stat card, that displays a count as well as a plot. No way to get the best of both?

**Reply by vopani** [reply]
The card clicker should work on any card type (including stat cards).

**Reply by n-srinidhi** [reply]
Thank you!
